.github/workflows/performance.yml

```diff
@@ -0,0 +1,7 @@
+- name: Run mobile performance testing script
+  run: ./gutenberg/bin/reassure-tests.sh
+
+- name: Report mobile performance results
+  run: npm danger ci
```

Comment:
Referenced in a previous comment, I do not believe this is a valid usage of the `npm` CLI. While I understand `yarn` is capable of executing CLI commands provided by packages, I believe `npm` requires that a script be defined in `package.json` first. 

```json
{
  "scripts": {
    "performance:report": "danger ci"
  }
}
```

```shell
npm run performance:report
```

Reply:
> While I understand yarn is capable of executing CLI commands provided by packages, I believe npm requires that a script be defined in package.json first.

Thanks for the tip. I wasn't aware of this distinction.